.travis.yml

```diff
@@ -46,6 +46,7 @@ env:
      - "TESTDIR=Java/curacao"
      - "TESTDIR=Java/dropwizard"
      - "TESTDIR=Java/gemini"
+     - "TESTDIR=Java/greenlightning
```

Comment:
Missing the closing quotation mark here, so this failed to run on travis.

Reply:
Drat, my bad, will fix tonight.